docs/source/common/lightning_module.rst

```diff
@@ -938,13 +983,12 @@ The current logger being used (tensorboard or other supported logger)
 
 local_rank
 ~~~~~~~~~~~
-The local_rank of this LightningModule. Lightning saves logs, weights etc only from global_rank = 0. You
-normally do not need to use this property
+The local_rank of this LightningModule. Lightning saves logs, weights etc. only from global_rank = 0. You
```

Comment:
```suggestion
The ``local_rank`` of this LightningModule. Lightning saves logs, weights etc. only from global_rank = 0. You
```

Reply:
I will reformulate this. We are talking about `local_rank` and global_rank is used as above. This is confusing.

Reply:
done!